WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -67,13 +71,23 @@ struct HubMenu: View {
         @ScaledMetric private var settingsIconSize: CGFloat = 20
 
         var body: some View {
-            HStack() {
+            HStack(spacing: Constants.padding) {
+                if let avatarURL = avatarURL {
+                    KFImage(avatarURL)
+                        .resizable()
+                        .clipShape(Circle())
+                        .frame(width: Constants.avatarSize, height: Constants.avatarSize)
```

Comment:
Since the avatar should be aligned top-left to the whole stack, I think it should be wrapped inside a VStack and pushed to the top with a `Spacer()`. WDYT?

Reply:
You are right, it's top-left aligned. 👍 fixed 

![Simulator Screen Shot - iPhone 13 Pro Max - 2021-12-24 at 15 09 49](https://user-images.githubusercontent.com/495617/147358212-bdd6bcd3-5990-49fc-8f8a-b28f153c2336.png)